andela/activo-api
# #158721928 Convert Response object to camel case

#### What does this PR do?
Converts all keys in the response object from `snake_case` to `camelCase` 

#### Description of Task to be completed?
Currently, the backend makes use of snake casing, while the frontend makes use of camel casing, so the code should be refactored to return camel case to the front-end while also converting the camel case received from the frontend to snake case.

#### How should this be manually tested?

- Fetch the branch `git fetch origin ch-convert-response-to-camel-case-158721928`
- Checkout to the new branch `git checkout ch-convert-response-to-camel-case-158721928`

- Run the application `Flask run`

- Go to any endpoint and check if the 'keys` of the response object is in `camelCase``

#### Any background context you want to provide?
We are using Marshmallow's `load_from` and `dump_to` functionalities

#### What are the relevant pivotal tracker stories?
[158721928](https://www.pivotaltracker.com/story/show/158721928)

#### Screenshots
**Before**
<img width="550" alt="screen shot 2018-07-05 at 15 07 22" src="https://user-images.githubusercontent.com/19773670/42444878-fa232e0e-8379-11e8-9342-751db0368b2b.png">

**After**
<img width="490" alt="screen shot 2018-07-06 at 21 05 54" src="https://user-images.githubusercontent.com/19773670/42444902-0b27219c-837a-11e8-9fab-4eb1c3b15ffb.png">
